packages/spaces/src/useLink.js

```diff
@@ -26,6 +26,8 @@ export default (spaceOrSpaceId, { clientId: propsClientId, linkAttributes = {} }
     window.open(`/web/spc/disclaimers/#/?spaceId=${id}&ssoText=${metadata.disclaimerId}`, '_self');
   };
 
+  const { data: user } = useCurrentUser();
```

Comment:
I know me and @jordan-a-young eventually want to move as many things as we can in `availity-react` and `sdk-js` to use `react-query`, but this will be a breaking change since users will need to wrap their app in a query context provider. I think for now let's get the user info with the [sdk-js resource](https://github.com/Availity/sdk-js/blob/master/packages/api-axios/src/resources/user.js#L12) and we can move this to our react-query hooks in the future.

AFAICT the rest of the SpacesLink changes have been made backwards compatible so far, so that's why I'm hesitant to introduce react-query here. What do you all think? If we add it here, then most of the fetching logic in `SpacesLink` could/should be rewritten to take advantage of it IMO, which is probably out of scope for this PR

Reply:
Yeah, that makes sense. However, the ticket I had specifically recommended moving to userCurrentUser(). Also, I do have a ticket in the epic to convert to react-query.

Reply:
We've decided that at least for the time-being we're going to move forward with the react-query solution.